DataStructures/HashMap/Hashing/Open_Addressing.java

```diff
@@ -0,0 +1,105 @@
+/**
+ * Here I imnplement a hashing function
+ * the Open_Addressing object uses an array for hashing
+ * using quadratic hashing, we make it possible to add an element and retrieve its value from the array in constant time
+ */
+public class Open_Addressing {
```

Comment:
It should be `OpenAddressing`, not `Open_Addressing`.

Reply:
Changed the name in my second commit